.gitignore

```diff
@@ -1,3 +1,5 @@
+maybe-you-can-help.md
+maybe-you-can-help.mdbak
```

Comment:
oops, some noise in the .gitignore.

Reply:
I just removed them.  Will repeat pull request.  tx.